Add dissenter extension app id to first party extension constants
--- a/chrome/common/extensions/extension_constants.cc
+++ b/chrome/common/extensions/extension_constants.cc
@@ -41,6 +41,7 @@ const char kGoogleSlidesAppId[] = "aapoc
 const char kIdentityApiUiAppId[] = "ahjaciijnoiaklcomgnblndopackapon";
 const char kTextEditorAppId[] = "mmfbcljfglbokpmkimbfghdkjmjhdgbg";
 const char kInAppPaymentsSupportAppId[] = "nmmhkkegccagdldgiimedpiccmgmieda";
+const char kDissenterAppId[] = "mhomeaadffipapncjcgmmcldklcoiigj";
 
 const char* const kBuiltInFirstPartyExtensionIds[] = {
     kCalculatorAppId,
@@ -65,6 +66,7 @@ const char* const kBuiltInFirstPartyExte
     kTextEditorAppId,
     kInAppPaymentsSupportAppId,
     kCastExtensionIdRelease,
+    kDissenterAppId,
 #if BUILDFLAG(IS_CHROMEOS_ASH)
     kAssessmentAssistantExtensionId,
     kAccessibilityCommonExtensionId,
